test/DartTest.ts

```diff
@@ -350,6 +350,7 @@ describe('transpile to dart', () => {
       expectTranslate('var x = {a: 1, b: 2}').to.equal(' var x = { "a" : 1 , "b" : 2 } ;');
       expectTranslate('var x = {a: 1, }').to.equal(' var x = { "a" : 1 } ;');
       expectTranslate('var x = {}').to.equal(' var x = { } ;');
+      expectTranslate('var x = {y}').to.equal(' var x = { "y" : y } ;');
```

Comment:
I don't think we should support object literals. The semantics are way too different. If you want to use a map in JS, use Map. Otherwise define a class. We already have a facade to work with maps. The transpiler should simply throw in this case.